docs/sources/k6/next/set-up/set-up-distributed-k6/install-k6-operator.md

```diff
@@ -50,27 +50,56 @@ You can find a complete list of Helm options in the [k6 Operator charts folder](
 
 ### Deploy with Makefile
 
-In order to install the operator with a Makefile, you'll need:
+This is a more manual, low-level way to install the k6 Operator. It installs from the branch of the k6-operator repository. By default, it's the `main` branch, which is not guaranteed to be always stable. In general case, deployment from the branch is meant for development purposes.
```

Comment:
```suggestion
This method installs the k6 Operator from the GitHub repository. By default, it uses the `main` branch, which is not guaranteed to be stable at all times. This method is helpful for:

- Development purposes.
- Users who have a `kustomize` pipeline and want to adjust manifests with it.
```

Reply:
:thinking: Strictly speaking, bundle method also installs from the Github repository... It just uses the file that is changing only on the releases and can be considered an artifact. IOW, it seems like there is a bit of a loss of information here. 

What about this version?
```suggestion
This method installs the k6 Operator from the GitHub repository, using the latest files in a branch. By default, it uses the `main` branch, which is not guaranteed to be stable at all times. This method is helpful for:
- Development purposes.
- Users who have a `kustomize` pipeline and want to adjust manifests with it.
```

Reply:
That looks good to me! 🙇